### ISSUE TITLE
MySQL install failed

### ISSUE THREAD

**Issue by wirehack7** [moritz/ilbot#35]
When I run *./install* I get this error:
```
Database connection is fine, creating the schema for you...
Schema creation failed: DBD::mysql::db do failed: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'DELIMITER //
-- no empty lines, would confuse the installer
CREATE PROCEDURE `il' at line 1 at ./install line 116, <$_[...]> chunk 8.
```

What to change there? The connection is working, see the first line.
I added the database user with ```GRANT ALL PRIVILEGES```.

**Comment by crypt1d** [comment]
I'm getting the same error with MariaDB on CentOS 7...

**Comment by moritz** [comment]
Should be fixed by 5604958098c482202cd41b21f370aca3b93bb0a8. Thanks for the
